src/main/java/com/crudtest/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.crudtest.repository;
+
+import com.crudtest.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+
+    User findByLastName(String lastName);
```

Comment:
Što se ovdje događa ako dva korsinika imaju isto prezime?

Reply:
da za sada je trebalo biti tako, ali cu to maknuti kad budem dodavao vise polja za formu kako je glasilo u onom prvom zadataku pa to mogu promijeniti da gleda npr po mailu ili usernamu, jer ce se naci uvijek da ljudi imaju ista prezimena pa je malo to sad glupo ispalo u ovom

Reply:
ok, također možeš imati sličnu metodu koja vraća nekakvu kolekciju. možda baš hoćeš sve ljude koje imaju usto prezime.
također za editiranje postojećeg korisnika je možda lakše da sakriješ unutarnji id korisnika negdje u formu pa ga tako dohvatiš ili definiraš još jedan uid (npr username ili email). uid - unique identifier.